### PR TITLE
Add support to create ldap identity provider

### PR DESCRIPTION
**Pull request by aloganat**
Signed-off-by: Arthy Loganathan <aloganat@redhat.com>

### CODE REVIEW

File: utils/scripts/openshift_cluster_manager/ldap.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+
+apiVersion: v1
```
[the remark this answers is not in the document]
Comment (aloganat) [reply utils/scripts/openshift_cluster_manager/ldap.yaml]: comments incorporated

File: utils/scripts/openshift_cluster_manager/openshift_cluster_manager.py [hunk 2]
```diff
@@ -267,34 +294,107 @@ def install_rhods(self):
...
+        else:
+            cmd = ("ocm create user {} --cluster {} "
+                   "--group={}".format(user,
+                                       self.cluster_name, group))
         ret = execute_command(cmd)
         if ret is None:
             print("Failed to add user {} to group "
-                  "{}".format(self.htpasswd_cluster_admin, group))
+                  "{}".format(user, group))
+            sys.exit(1)
```
[the remark this answers is not in the document]
Comment (aloganat) [reply utils/scripts/openshift_cluster_manager/openshift_cluster_manager.py:368]: comments incorporated

File: utils/scripts/ocm/ocm.py [hunk 3]
```diff
@@ -267,34 +294,119 @@ def install_rhods(self):
...
+                                   group="rhods-admins")
+
+        # Adds user ldap-user1..ldap-user20
+        for i in range(1, 21):
+            self.add_user_to_group(user="ldap-users"+str(i),
+                                   group="rhods-users")
+
+        self.create_group("rhods-noaccess")
+        # Adds user ldap-noaccess1..ldap-noaccess20
+        for i in range(1, 21):
```
Comment (LaVLaS) [inline utils/scripts/ocm/ocm.py:406]: I would recommend using a argument variable for the end of the range (`21`) to allow the ability change it dynamically

Comment (aloganat) [reply utils/scripts/ocm/ocm.py]: comments incorporated.

**Review by vasukulkarni — APPROVED** [review]

**Pull request merged**